src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -87,9 +90,11 @@ public void Initialize(IEventSource eventSource)
         {
             _initialTargetOutputLogging = Environment.GetEnvironmentVariable("MSBUILDTARGETOUTPUTLOGGING");
             _initialLogImports = Environment.GetEnvironmentVariable("MSBUILDLOGIMPORTS");
+            _initialTrackProperties = Environment.GetEnvironmentVariable("MsBuildLogPropertyTracking");
 
             Environment.SetEnvironmentVariable("MSBUILDTARGETOUTPUTLOGGING", "true");
             Environment.SetEnvironmentVariable("MSBUILDLOGIMPORTS", "1");
+            Environment.SetEnvironmentVariable("MsBuildLogPropertyTracking", "1");
```

Comment:
This worries me; I don't want to slow binary logging down too much, even though this is valuable information. I don't want to give anyone an excuse to not turn binlog on. Do we have a clear picture of the impact?

Reply:
This was removed.